assets/src/domain/wpPluginsPage/ExitModal.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef } from 'react';
+import { pathOr } from 'ramda';
+import { addQueryArgs } from '@wordpress/url';
+import * as typeformEmbed from '@typeform/embed';
+
+import { ExitModalInfo } from './types';
+import { Modal } from '@wordpress/components';
+import './styles.scss';
+
+type ExitModalProps = {
+	onSubmit: VoidFunction;
+	isOpen: boolean;
+};
+
+const info = pathOr<ExitModalInfo>(null, ['eejsdata', 'data', 'exitModalInfo'], window);
+
+const ModalContent: React.FC<Partial<ExitModalProps>> = ({ onSubmit }) => {
+	const typeFormEl = useRef();
+	const typeFormUrl = addQueryArgs('https://eventespresso.typeform.com/to/O1DDym', {
```

Comment:
shouldn't this be in `.env`?
https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
I agree but it was there in the existing code.

Reply:
Tried, it doesn't work, may be something missing in webpack config.